Name: Hide SprintTV app
Version: 1.4.0-1
Author: noir, cryptk
Description: This will hide the SprintTV app in the App Launcher.

--- .orig/usr/palm/applications/com.mobitv.app.sprinttv/appinfo.json
+++ /usr/palm/applications/com.mobitv.app.sprinttv/appinfo.json
@@ -9,5 +9,6 @@
 	"vendor": "Sprint",
 	"vendorurl": "www.sprint.com",
 	"theme": "light",
-	"noDeprecatedStyles": false
+	"noDeprecatedStyles": false,
+	"visible": false
 }
